packages/nc-gui-v2/components/template/Editor.vue

```diff
@@ -174,9 +179,8 @@ async function importTemplate() {
   try {
     await validate()
   } catch (errorInfo) {
-    toast.error('Please fill all the required values')
     isImporting.value = false
-    return
+    throw new Error('Please fill all the required values')
```

Comment:
why an error suddenly?

Reply:
`importTemplate` in `Editor.vue` is called by `handleImport` in `QuickImport.vue`. Throwing in `Editor.vue` so that `QuickImport.vue` could catch it.